packages/wonder-blocks-button/src/components/button-core.js

```diff
@@ -100,11 +100,13 @@ export default class ButtonCore extends React.Component<Props> {
 
         const Label = size === "small" ? LabelSmall : LabelLarge;
 
+        const iconSize = size === "small" ? "small" : "medium";
```

Comment:
Could you add a comment to the code explaining why the icon sizes don't line up exactly with the button sizes?